### PR TITLE
Release/v1.9.301 to develop

### PR DESCRIPTION
**Pull request by pcm32**
For reviewing purposes of the release changes only.

### CODE REVIEW

**Comment by pcm32** [comment]
Please do not merge here, I will merge in sourcetree locally and push once this is approved.

**Comment by pmb59** [comment]
HI @pcm32, why do you need `scipy` and  `scikit-learn` in this release?

**Comment by pcm32** [comment]
It is part of the pinning done to have everything working with scanpy 1.9.x (tests passing). Those changes are part of previous reviewed PRs, what needs approval here is more the formal release part (ie. that all parts where the scanpy-scripts package version should be changed are changed, the version scheme, etc). Thanks for taking a look.

**Comment by pcm32** [comment]
To be more explicit, those have always been dependencies of scanpy scripts, but latest versions broke bknn, so we had to add them explicitly with version constraints.

File: setup.py [hunk 1]
```diff
@@ -49,8 +49,10 @@
         "Click<8",
         # "umap-learn",
         "harmonypy>=0.0.5",
-        "bbknn>=1.5.0",
+        "bbknn>=1.5.0,<1.6.0",
         "mnnpy>=0.1.9.5",
+        "scipy<1.9.0",
+        "scikit-learn<1.3.0",
         "scrublet",
         # "scikit-misc",
```
Comment (pmb59) [inline setup.py:57]:
```suggestion
```
There are some commented-out dependencies in the install_requires list, which may be unnecessary

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ A command-line interface for functions of the Scanpy suite, to facilitate flexib
 
 ## Install
 
-The recommended way of installing scanpy-scripts is via conda:
+The recommended way of using this package is through the latest container produced by Bioconda [here](https://quay.io/repository/biocontainers/scanpy-scripts?tab=tags). If you must, prefer installing scanpy-scripts is via conda:
```
Comment (pmb59) [inline README.md:7]:
```suggestion
The recommended way of using this package is through the latest container produced by Bioconda [here](https://quay.io/repository/biocontainers/scanpy-scripts?tab=tags). If you must, one can install scanpy-scripts via conda:
```

File: README.md [hunk 3]
```diff
@@ -63,3 +63,7 @@ Commands:
   multiplet  Execute methods for multiplet removal.
   plot       Visualise data.
   ```
+
+  ## Versioning
+
+  Major and major versions will follow the scanpy versions. The first digit of the patch should follow the scanpy patch version as well, subsequent digits in the patch are reserved for changes in this repository.
```
Comment (pmb59) [inline README.md:69]: Not very clear where the `.301` comes from, in this release.

**Review by pmb59 — APPROVED** [review]
LGTM @pcm32

**Comment by pcm32** [comment]
Thanks for the review @pmb59 .

**Pull request merged**